tools/stimulus/types-RR.yaml

```diff
@@ -546,3 +546,7 @@ VERTEX_WEIGHTS:
         } else {
           %I% <- NULL
         }
+
+MSTALGORITHM:
+    DEFAULT:
+        AUTOMATIC: c("automatic", "prim", "kruskal", "unweighted")
```

Comment:
This is technically sufficient, but I strongly recommend setting up all values (`PRIM`, `KRUSKAL`, etc.), otherwise someone will get confused in the future when _just for this one type_ (`MSTALGORITHM`) only _some_ values work, while for other types all values work.